packages/plugins/typescript-apollo-angular/package.json

```diff
@@ -10,13 +10,16 @@
     "test": "codegen-templates-scripts test",
     "pretest": "yarn build"
   },
+  "peerDependencies": {
+    "graphql-tag": "^2.0.0"
+  },
   "dependencies": {
     "dependency-graph": "0.7.2",
+    "graphql-codegen-plugin-helpers": "0.13.0",
```

Comment:
this plugin is missing in all plugins because webpack treats it as an external module but it was a devDep, so not included in the project.